website/docs/r/schema_site_anp_epg.html.markdown

```diff
@@ -29,6 +29,7 @@ resource "mso_schema_site_anp_epg" "site_anp_epg" {
 * `site_id` - (Required) SiteID under which you want to deploy Anp Epg.
 * `anp_name` - (Required) Name of Application Network Profiles.
 * `epg_name` - (Required) Name of Endpoint Group to manage.
+* `private_link_label` - (Optional) Private Link Label. This argument is required when the EPG type is service at template level.
```

Comment:
It is not always required. Only on certain cases if the type of service is private. Remove the second sentence.

Reply:
ok